Author: DoloresPark <dolorespark@e-vertise.com>
Date:   Sun Mar 02 22:13:30 2014 -0500

    fix fast audio
    
    add missing symbols that prevent libnvasfparserhal.so and
    libnvaviparserhal.so from loading; also, remove access to
    Nvidia's HE-AAC+ codec which remains broken

<<<<<<< HEAD
diff --git a/frameworks/av/include/media/stagefright/MediaDefs.h b/frameworks/av/include/media/stagefright/MediaDefs.h
=======
diff --git a/frameworks/av/include/media/stagefright/MediaDefs.h b/frameworks/av/include/stagefright/MediaDefs.h
>>>>>>> 568ea642b03dc73560abc44688a46c7a1fabd7dd
index 85693d4..fcbca98 100644
--- a/frameworks/av/include/media/stagefright/MediaDefs.h
+++ b/frameworks/av/include/media/stagefright/MediaDefs.h
@@ -20,6 +20,8 @@
 
 namespace android {
 
+// DoPa (2014/02/26) - added codecs needed by Nvidia blobs
+
 extern const char *MEDIA_MIMETYPE_IMAGE_JPEG;
 
 extern const char *MEDIA_MIMETYPE_VIDEO_VP8;
@@ -29,6 +31,8 @@ extern const char *MEDIA_MIMETYPE_VIDEO_MPEG4;
 extern const char *MEDIA_MIMETYPE_VIDEO_H263;
 extern const char *MEDIA_MIMETYPE_VIDEO_MPEG2;
 extern const char *MEDIA_MIMETYPE_VIDEO_RAW;
+extern const char *MEDIA_MIMETYPE_VIDEO_WMV;    // Nvidia
+extern const char *MEDIA_MIMETYPE_VIDEO_MJPEG;  // Nvidia
 
 extern const char *MEDIA_MIMETYPE_AUDIO_AMR_NB;
 extern const char *MEDIA_MIMETYPE_AUDIO_AMR_WB;
@@ -44,6 +48,8 @@ extern const char *MEDIA_MIMETYPE_AUDIO_RAW;
 extern const char *MEDIA_MIMETYPE_AUDIO_FLAC;
 extern const char *MEDIA_MIMETYPE_AUDIO_AAC_ADTS;
 extern const char *MEDIA_MIMETYPE_AUDIO_MSGSM;
+extern const char *MEDIA_MIMETYPE_AUDIO_WMA;    // Nvidia
+extern const char *MEDIA_MIMETYPE_AUDIO_AC3;    // Nvidia
 
 extern const char *MEDIA_MIMETYPE_CONTAINER_MPEG4;
 extern const char *MEDIA_MIMETYPE_CONTAINER_WAV;
@@ -54,6 +60,7 @@ extern const char *MEDIA_MIMETYPE_CONTAINER_AVI;
 extern const char *MEDIA_MIMETYPE_CONTAINER_MPEG2PS;
 
 extern const char *MEDIA_MIMETYPE_CONTAINER_WVM;
+extern const char *MEDIA_MIMETYPE_CONTAINER_ASF;    // Nvidia
 
 extern const char *MEDIA_MIMETYPE_TEXT_3GPP;
 extern const char *MEDIA_MIMETYPE_TEXT_SUBRIP;

